src/test/java/com/google/dicomwebfuse/fuse/DicomFuseHelperIT.java

```diff
@@ -0,0 +1,213 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.dicomwebfuse.fuse;
+
+import static com.google.dicomwebfuse.EntityType.DICOM_STORE;
+import static com.google.dicomwebfuse.EntityType.INSTANCE;
+import static com.google.dicomwebfuse.EntityType.SERIES;
+import static com.google.dicomwebfuse.EntityType.STUDY;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.google.api.client.http.HttpStatusCodes;
+import com.google.dicomwebfuse.TestUtils;
+import com.google.dicomwebfuse.auth.AuthAdc;
+import com.google.dicomwebfuse.dao.FuseDao;
+import com.google.dicomwebfuse.dao.FuseDaoImpl;
+import com.google.dicomwebfuse.dao.http.HttpClientFactory;
+import com.google.dicomwebfuse.entities.CloudConf;
+import com.google.dicomwebfuse.entities.DicomPath;
+import com.google.dicomwebfuse.entities.cache.Cache;
+import com.google.dicomwebfuse.exception.DicomFuseException;
+import com.google.dicomwebfuse.fuse.cacher.DicomPathCacher;
+import com.google.dicomwebfuse.parser.Arguments;
+import java.io.IOException;
+import jnr.ffi.Platform;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+class DicomFuseHelperIT {
```

Comment:
Let's keep this named DicomFuseHelperTest... anything that doesn't require any external resources I consider a unit test.

Reply:
Done